ktx-library-codelab/step-03/myktxlibrary/src/main/java/com/example/android/myktxlibrary/LocationUtils.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.android.myktxlibrary
+
+import android.location.Location
+import com.google.android.gms.location.LocationRequest
+
+fun createLocationRequest() = LocationRequest.create()!!.apply {
```

Comment:
Nit: `?` instead of `!!`?

Reply:
removing it, this is a platform type which is missing an annotation, but can never be `null`